code/java/index1.java

```diff
@@ -0,0 +1,103 @@
+package code.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.xapian.Document;
+import org.xapian.Stem;
+import org.xapian.TermGenerator;
+import org.xapian.WritableDatabase;
+import org.xapian.XapianConstants;
+import org.xapian.XapianJNI;
+
+public class index1 {
+    
+    // Command line args - dbpath datapath
+    public static void main(String[] args)
+    {
+        if(args.length < 2)
```

Comment:
:+1: not a problem — it takes a while to get used to any project's formatting conventions :-)